publication-commons/src/main/java/no/unit/nva/publication/model/business/importcandidate/ImportCandidate.java

```diff
@@ -109,7 +109,7 @@ public void setEntityDescription(EntityDescription entityDescription) {
     }
 
     public AssociatedArtifactList getAssociatedArtifacts() {
-        return associatedArtifacts;
+        return nonNull(associatedArtifacts) ? associatedArtifacts : AssociatedArtifactList.empty();
```

Comment:
POLA: If not already true, probably make the associatedArtifacts unmodifiable to match emptySet